### PR TITLE
Add flags for updated api calls in upcoming tsuru

### PR DESCRIPTION
**Pull request by cezarsa**
This includes new app-update flags and new role-assign arguments.

### CODE REVIEW

File: tsuru/client/apps.go [hunk 1]
```diff
@@ -210,70 +214,50 @@ func (c *AppCreate) Run(context *cmd.Context, client *cmd.Client) error {
...
-
-The [[--team-owner]] parameter sets owner team for an application.
-
-The [[--platform]] parameter sets a platform for an application.
-
-The [[--image-reset]] parameter rebuilds the platform of an application.
-
-The [[--tag]] parameter sets a tag for your app. You can set
-multiple [[--tag]] parameters.`,
+		Usage: "app-update [-a/--app appname] [--description/-d description] [--plan/-p plan name] [--pool/-o pool] [--team-owner/-t team owner] [--platform/-l platform] [-i/--image-reset] [--cpu cpu] [--memory memory] [--tag/-g tag]...",
```
Comment (wpjunior) [inline tsuru/client/apps.go:228]: @cezarsa I miss whats the format of --memory, "500mb", "500M" or "500Mib" ?
also for `--cpu` param.

File: tsuru/installer/dm/dockermachine.go [hunk 2]
```diff
@@ -200,10 +200,6 @@ func (d *DockerMachine) createRegistryCertificate(hosts ...string) error {
 	return generator.GenerateCert(certOpts)
 }
 
-func (d *DockerMachine) DeleteAll() error {
```
Comment (wpjunior) [inline tsuru/installer/dm/dockermachine.go:203]: @cezarsa I didn't understand why we removed that.

Comment (cezarsa) [reply tsuru/installer/dm/dockermachine.go:203]: It wasn't called anywhere. I found it running `staticcheck -unused.whole-program` and filtering the false positives due to test code.

File: tsuru/client/apps.go (hunk 1 above)
Comment (cezarsa) [reply tsuru/client/apps.go:228]: We're using kubernetes resource lib to parse this value, I'll add a description including the supported suffixes.

Comment (cezarsa) [reply tsuru/client/apps.go:228]: Done

**Pull request merged**